### DISCUSSION TITLE
FATAL - Unsupported Elasticsearch version: 5.6.12. Aborting.

### DISCUSSION THREAD

**Discussion by mark8s** [jertel/elastalert2#1524] [Q&A] [answered]
My company's es version is 5.x, and the alarm error with elastalert2 is `FATAL - Unsupported Elasticsearch version: 5.6.12. Aborting.`, but the company's es is not easy to upgrade because it stores a lot of production log data, and the upgrade may be risky. Is there any way to solve this problem for me?

**Comment by jertel** [comment, marked as answer]
I would greatly appreciate if you would please read the project guidelines before posting. Specifically, do not create issues for questions. Use discussions.

Are you planning on staying on an unsupported software version permanently? If so you will need to find a very old (ancient) version of ElastAlert that is compatible with that old version of Elasticsearch.

**Reply by mark8s** [reply]
@mark8s ok,tks~
